README.md

```diff
@@ -27,20 +27,25 @@ kubectl -n kube-system get deployment coredns -o yaml | \
 
 ### OpenShift
 Follow basic setup [here](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started)
+Use oc from https://apb-oc.s3.amazonaws.com/apb-oc/oc-linux-64bit.tar.gz to be able to correctly deploy openshift 3.11 alpha.
 
 ```bash
-oc cluster up --tag=v3.11
+oc cluster up --tag=v3.11 --public-hostname=$(hostname -f)
```

Comment:
If DNS isn't setup on the hostname, then it will resolve to localhost.  Might be better to use nic's IP instead.
```bash
IP=$(ip addr show eth0 | grep -Po 'inet \K[\d.]+')
oc cluster up --tag=v3.11 --public-hostname=${IP}
```